CHANGELOG.md

```diff
@@ -87,6 +87,7 @@
 - Bugfix: Fix directory not opening when clicking "Open AppData Directory" setting button on macOS (#2531, #2537)
 - Bugfix: Fix quickswitcher not respecting order of tabs when filtering (#2519, #2561)
 - Bugfix: Fix GNOME not associating Chatterino's window with its desktop entry (#1863, #2587)
+- Bugfix: Fix bold/coloured usernames also formatting trailing punctuation (#2593, #2597)
```

Comment:
Please add this PR number to the `- Major: Color mentions to match the mentioned users. You can disable this by unchecking "Color @usernames" under `Settings -> General -> Advanced (misc.)`. (#1963, #2284)` entry instead of adding a new entry

Reply:
Modified.